kyo-scheduler/jvm/src/main/scala/kyo/scheduler/InternalClock.scala

```diff
@@ -4,7 +4,7 @@ import java.util.concurrent.Executor
 import java.util.concurrent.locks.LockSupport
 import kyo.stats.internal.MetricReceiver
 
-final private case class InternalClock(executor: Executor) {
+final case class InternalClock(executor: Executor) {
```

Comment:
As you mentioned in description, I think you can make both package private.

Reply:
Will do

Reply:
Is this still pending?

Reply:
done